merkletree/pad.go

```diff
@@ -5,6 +5,7 @@ import (
 	"errors"
 
 	"github.com/coniks-sys/coniks-go/crypto"
+	conikshasher "github.com/coniks-sys/coniks-go/crypto/hasher/coniks"
```

Comment:
I would rather prefer a shorter import alias. What do you think about `hasher`? As this whole project is called 'coniks', I feel `hasher` would be enough.